core/rawdb/freezer_table_test.go

```diff
@@ -692,3 +692,23 @@ func TestAppendTruncateParallel(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkFreezerTable(b *testing.B) {
+	dir := "./tmp-deleteme"
+	defer os.RemoveAll(dir)
+	f, err := newCustomTable(dir, "foobar", metrics.NilMeter{}, metrics.NilMeter{}, metrics.NilGauge{}, 20*1024*1024, true)
+	if err != nil {
+		b.Fatal(err)
+	}
+	data := make([]byte, 32)
+	rand.Read(data)
+	b.ResetTimer()
+	b.ReportAllocs()
+	b.SetBytes(int64(len(data) * b.N))
```

Comment:
This is not correct. The `SetBytes` argument is the data amount per iteration, not the whole data amount.

Reply:
Also, you should make a benchmark with slightly larger data amount.

Reply:
Ah, good, that makes more sense! As for larger data, I wanted to have it at the worst case, but sure, more is better